core/domain/takeout_domain.py

```diff
@@ -0,0 +1,70 @@
+# coding: utf-8
+#
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.]
+
+"""Domain objects for Takeout."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import python_utils
+
+
+class TakeoutData(python_utils.OBJECT):
+    """Domain object for all information exported as part of Takeout."""
+
+    def __init__(self, user_data, user_images):
+        """Constructs a TakeoutDomain domain object.
+
+        Args:
+            user_data: dict. User's Takeout data stored as a dictionary.
+            user_images: list. List of TakeoutImage objects representing user's
```

Comment:
list(...) (please include typeinfo of each element in the list)

Reply:
Done